CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Improvements
 
 -   Enable configuring `ResourceOptions` via `transformations` (https://github.com/pulumi/pulumi-kubernetes/pull/575).
+-   Implement CheckConfig/DiffConfig (https://github.com/pulumi/pulumi-kubernetes/pull/577).
```

Comment:
Could we rephrase this in terms of something an end user would understanding (like "changing configuration for the kubernetes provider will now cause resource it manages to be replaced" or something?)